Sources/TuistKit/Utils/CommentsRemover.swift

```diff
@@ -0,0 +1,16 @@
+import Foundation
+
+enum CommentsRemover {
+    static func removeComments(from code: String) throws -> String {
+        let regexPattern = #"//.*?$|/\*[\s\S]*?\*/"#
+        do {
+            let regex = try NSRegularExpression(pattern: regexPattern, options: [.anchorsMatchLines])
+            let range = NSRange(location: 0, length: code.utf16.count)
+            var cleanCode = regex.stringByReplacingMatches(in: code, options: [], range: range, withTemplate: "")
+
+            cleanCode = cleanCode.replacingOccurrences(of: #"(?m)^\s*\n"#, with: "", options: .regularExpression)
+
+            return cleanCode
+        }
+    }
+}
```

Comment:
I'd make this a private method in `ImportSourceCodeScanner`.

If we expect this code to be shared, it belongs to `String+Extras.swift` instead

Reply:
Do you want to remove CommentsRemoverTests?

Reply:
If we're gonna make the method private, then yes.

If you think this is suited to put into `String+Extras.swift`, then we can keep them. I'll leave that decision to you, I don't feel strongly about this.